comparison/loadtest.go

```diff
@@ -247,13 +313,25 @@ func initLoadTest(t *terraform.Terraform, buildCfg BuildConfig, dumpFilename str
 	defer resetBucketCancel()
 	resetBucketErrCh := make(chan error, 1)
 	go func() {
-		for _, c := range resetBucketCmds {
-			mlog.Info(c.msg)
-			if err := exec.CommandContext(resetBucketCtx, "aws", c.value...).Run(); err != nil {
-				resetBucketErrCh <- fmt.Errorf("failed to run local cmd %q: %w", c.value, err)
+		if s3BucketURI != "" && tfOutput.HasS3Bucket() {
+
+			mlog.Info("Emptying S3 bucket")
```

Comment:
We can remove this empty line
```suggestion
		if s3BucketURI != "" && tfOutput.HasS3Bucket() {
			mlog.Info("Emptying S3 bucket")
```